packages/sit-onyx/src/components/OnyxFAB/OnyxFAB.vue

```diff
@@ -115,7 +134,7 @@ const triggerIcon = computed(() => {
 
     &--left {
       right: unset;
-      left: var(--onyx-fab-viewport-gap);
+      left: var(--onyx-fab-offset-x);
```

Comment:
```suggestion
    --onyx-fab-viewport-gap: var(--onyx-density-sm);
    --onyx-fab-offset-x: 0;
    --onyx-fab-offset-y: 0;
    --onyx-basic-popover-gap: var(--onyx-density-sm);

    font-family: var(--onyx-font-family);
    color: var(--onyx-color-text-icons-neutral-inverted);
    position: fixed;
    bottom: calc(var(--onyx-fab-viewport-gap) + var(--onyx-fab-offset-y));
    right: calc(var(--onyx-fab-viewport-gap) + var(--onyx-fab-offset-x));
    z-index: var(--onyx-z-index-notification);

    .onyx-flyout-menu__list-header,
    .onyx-flyout-menu__list-footer {
      display: none;
    }

    .onyx-flyout-menu__wrapper {
      display: flex;
      flex-direction: column;
      gap: var(--onyx-density-2xs);
      align-items: flex-end;
    }

    &--left {
      right: unset;
      left: calc(var(--onyx-fab-viewport-gap) + var(--onyx-fab-offset-x));
```

We should enforce the min viewport gap so the offset variables are just additional offsets on top of the viewport gap :)